data/10/measure_annotations.xml

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mei
+    xmlns="http://www.music-encoding.org/ns/mei">
+    <meiHead>
+        <fileDesc>
+            <titleStmt>
+                <title>Measure annotations for Col pianto e coi (Aria)</title>
+            </titleStmt>
+            <pubStmt/>
+        </fileDesc>
+        <encodingDesc>
+            <appInfo>
+                <application isodate="2019-11-05T13:51:46.916Z" version="1.0.2">
+                    <name>Deep Optical Measure Detector</name>
+                    <p>Measures detected with Deep Optical Measure Detector</p>
+                </application>
+            </appInfo>
+        </encodingDesc>
+    </meiHead>
+    <music>
+        <facsimile>
+            <surface xml:id="surface_01DRXXBW793Y78RN1NJJH4NB9M" n="1" ulx="0" uly="0" lrx="1997" lry="2476">
+                <graphic xml:id="graphic_01DRXXBW7CCJ95KSPMFMF07877" target="bsb10598495_00440.jpg" width="1998" height="2477"/>
```

Comment:
Shouldn't we be targeting to a specific canvas within an IIIF manifest, or to an IIIF Image API URI? Instead of just to a file? I'd like the facsimile files to be removed eventually (see the issue I filed)

Reply:
That is the question. We have multiple possibilities here: we could simply use a file name, even if we don't provide the file itself (probably we would keep the document URN and the manifest URL somewhere up in the header). We could point to the full graphic as delivered via IIIF, and so on. We could even provide multiple pointers in graphic with `@corresp` and use `@targettype` to specify what sort of URI is kept in the `@target` attribute. 
I don't have a clear preference.